shared/elements/gaia_lowstorage_dialog/style.css

```diff
@@ -0,0 +1,33 @@
+gaia-lowstorage-dialog[hidden] {
+  /* The underlying dialog is controlled by this attribute, but we don't want to
+   * hide the outer element so that we see the transition */
+  display: initial;
+}
+
+.button-link-like {
```

Comment:
__issue:__ ":active" state looks wrong for this link-like button (tap and hold a bit)